airbyte-integrations/connectors/source-salesforce/Dockerfile

```diff
@@ -1,13 +1,26 @@
-FROM python:3.7-slim
+FROM python:3.7.11-alpine3.14 as base
+FROM base as builder
 
-# Bash is installed for more convenient debugging.
-RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
+
+RUN apk --no-cache upgrade \
+    && pip install --upgrade pip  \
+    && apk --no-cache add tzdata build-base
 
 WORKDIR /airbyte/integration_code
-COPY source_salesforce ./source_salesforce
-COPY main.py ./
 COPY setup.py ./
-RUN pip install .
+RUN pip install --prefix=/install .
+
+
+FROM base
+COPY --from=builder /install /usr/local
+# add default timezone settings
+COPY --from=builder /usr/share/zoneinfo/Etc/UTC /etc/localtime
+RUN echo "Etc/UTC" > /etc/timezone
+
+WORKDIR /airbyte/integration_code
+COPY main.py ./
+COPY source_salesforce ./source_salesforce
+
```

Comment:
shouldn't we also change the dockerfile tag here?

Reply:
ok